.github/workflows/build.yml

```diff
@@ -22,32 +22,54 @@ jobs:
       - run: tox -e ${{ matrix.tox-job }}
   build:
     runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }} - ${{ matrix.python }} ${{ matrix.build }}
```

Comment:
The PR title is only about "env var", but these changes look unrelated to env vars. You might merge them in a separated PR.

Reply:
Done